ister.py

```diff
@@ -412,9 +412,17 @@ def get_current_format():
 def copy_os(args, template, target_dir):
     """Wrapper for running install command
     """
-    LOG.info("Starting swupd. May take several minutes")
-    add_bundles(template, target_dir)
+    package_manager = template["SoftwareManager"]
```

Comment:
Should update unit test to call out specificly running swupd and add test for dnf.